notebooks/MAST/HSC/CASJOBS/HCV_CASJOBS/HCV_casjobs_demo.ipynb

```diff
@@ -0,0 +1,995 @@
+    "* The column `VarQualFlag` is a variability quality flag (see Section 5 of the paper). The five letters correspond to CI, D, MagerrAper2, MagAper2-MagAuto, p2p; AAAAA corresponds to the highest quality flag.\n",
+    "* The column `FilterDetFlag` is the filter detection flag: 1=source is variable in this filter, 0=source is not variable in this filter.\n",
+    "\n",
+    "See the HCV paper by [Bonanos et al. (2019, AAp)](https://www.aanda.org/component/article?access=doi&doi=10.1051/0004-6361/201936026) for more details on the computation and meaning of these quantities."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Find objects with measurements in both F475W and F814W\n",
+    "\n",
+    "This could also be done in the SQL query.  Here we use the `Astropy.table.join` function instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w475 = np.where(tab['Filter'] == 'ACS_F475W')\n",
+    "w814 = np.where(tab['Filter'] == 'ACS_F814W')\n",
+    "\n",
+    "# the only key needed to do the join is MatchID, but we include other common columns\n",
+    "# so that join includes only one copy of them\n",
+    "jtab = join(tab[w475], tab[w814],\n",
+    "            keys=['MatchID', 'GroupID', 'SubGroupID', 'RA', 'Dec', 'AutoClass', 'ExpertClass'],\n",
+    "            table_names=['f475', 'f814'])\n",
+    "print(len(jtab), \"matched F475W+F814W objects\")\n",
+    "jtab[jtab['AutoClass'] > 0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot object positions on the sky <a name=\"positions\"></a>\n",
+    "\n",
+    "We mark the galaxy center as well.  Note that this field is in the outskirts of IC 1613.  The 0.5 degree search radius (which is the maximum allowed in the API) allows finding these objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(1, (10, 10))\n",
+    "plt.plot(jtab['RA'], jtab['Dec'], 'bo', markersize=1, label=f'{len(tab):,} HCV measurements')\n",
+    "plt.plot(ra, dec, 'rx', label=target, markersize=10)\n",
+    "plt.gca().invert_xaxis()\n",
+    "plt.gca().set_aspect('equal')\n",
+    "plt.xlabel('RA [deg]')\n",
+    "plt.ylabel('Dec [deg]')\n",
+    "plt.legend(loc='best')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot HCV MAD variability index versus magnitude in F475W <a name=\"variability\"></a>\n",
+    "\n",
+    "The median absolute deviation variability index is used by the HCV to identify variables.  It measures the scatter among the multi-epoch measurements.  Some scatter is expected from noise (which increases for fainter objects).   Objects with MAD values that are high are likely to be variable.\n",
```

Comment:
I really like this paragraph here.  It's clear and to the point but justifies what comes next. No change needed, just wanted to give positive feedback.